frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/image/custom_image_block_component.dart

```diff
@@ -355,27 +355,28 @@ class CustomImageBlockComponentState extends State<CustomImageBlockComponent>
 
   // only used on mobile platform
   List<Widget> _buildExtendActionWidgets(BuildContext context) {
-    final url = widget.node.attributes[CustomImageBlockKeys.url];
+    final String url = widget.node.attributes[CustomImageBlockKeys.url];
     if (!_checkIfURLIsValid(url)) {
       return [];
     }
 
     return [
-      FlowyOptionTile.text(
-        showTopBorder: false,
-        text: LocaleKeys.editor_copyLink.tr(),
-        leftIcon: const FlowySvg(
-          FlowySvgs.m_field_copy_s,
+      if (!url.contains('appflowy.cloud'))
```

Comment:
`startsWith` over contains, eg. if someone uploaded an image named `appflowy.cloud.image.png` to local, it would now contain this string.

Reply:
Can consider using a regexp to also match beta: `https?:\/\/([a-z0-9]+[.])*appflowy[.]cloud`